### PR TITLE
[16.0][MIG] currency_rate_update_xe: Migration to 16.0

### PR DESCRIPTION
**Pull request by mpascuall**
Module migrated to version 16.0

cc https://github.com/APSL 153469

@miquelalzanillas @lbarry-apsl @javierobcn @peluko00 please review

### CODE REVIEW

**Review by peluko00 — APPROVED** [review]
Functional review ok

**Review by javierobcn — APPROVED** [review]
Functional ok , checked in runboat

File: currency_rate_update_xe/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Currency Rate Update: XE.com",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
```
Comment (pedrobaeza) [inline currency_rate_update_xe/__manifest__.py:6]: This change (and the corresponding README modifications) should be done on the migration commit, not in the pre-commit auto-fixes.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: currency_rate_update_xe/__manifest__.py (hunk 1 above)
Comment (miquelpascual) [reply currency_rate_update_xe/__manifest__.py:6]: Fixed, please review

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**